### PR TITLE
jdbcio write batch config changes

### PR DESCRIPTION
**Pull request by mravi**
Addresses https://github.com/apache/beam/issues/32891

### CODE REVIEW

**Comment by mravi** [comment]
@chamikaramj @ahmedabu98 @Polber ptal

**Comment by mravi** [comment]
R: @ahmedabu98 @chamikaramj @Polber

File: sdks/python/apache_beam/io/external/xlang_jdbcio_it_test.py [hunk 1]
```diff
@@ -191,6 +191,7 @@ def test_xlang_jdbc_write_read(self, database):
               username=self.username,
               password=self.password,
               classpath=classpath,
+              write_batch_size=5,
```
Comment (Abacn) [inline sdks/python/apache_beam/io/external/xlang_jdbcio_it_test.py:194]: can we remove this additional config? This is to test the IO works with minimum configuration provided, and sometimes user refers to the existing test to write their pipeline

**Review by Abacn — COMMENTED** [review]
Thanks, this looks good to me. Just had one comment

**Pull request merged**